File: conditional/old_enough.py
# Zoey Sosa Old Enough python

age = int(input("How old are you?:\n"))


if age >=18:
   print("You can vote!")
elif age <18 and age >= 16:
    print("You can drive!")
elif age <16 and age >= 15:
    print("You can get a learners permit")
elif age >= 4:
    print("You can go to school!")
else:
     print("too bad you aren't any of these ages")

File: conditional/old_enough.c
//zoey sosa, Old Enough C
#include <stdio.h>
int num;

int main(void){
    printf("How old are you?\n");
    scanf("%d", &num);
if(num == 18){
    printf("you can vote\n");
}else if (num ==16){
    printf("you can drive!\n");
}else if (num == 15){
    printf("You can get a learners permit\n");
}else if (num ==4)
    printf("you can go to school\n");
else{
    printf("you aren't any of these ages\n");
}
return 0;
}